vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/ui/fragment/save/SaveCardFragment.kt

```diff
@@ -46,13 +50,16 @@ import com.verygoodsecurity.vgscheckout.util.command.save.SaveCardInfo
 import com.verygoodsecurity.vgscheckout.util.country.model.Country
 import com.verygoodsecurity.vgscheckout.util.country.model.PostalCodeType
 import com.verygoodsecurity.vgscheckout.util.extension.*
+import com.verygoodsecurity.vgscheckout.util.logger.VGSCheckoutLogger
 
 internal class SaveCardFragment : BaseFragment<CheckoutConfig>(),
     VGSCountryEditText.OnCountrySelectedListener, InputFieldView.OnEditorActionListener {
 
     private lateinit var binding: SaveCardViewBindingHelper
     private lateinit var validationHelper: ValidationManager
 
+    private val inputFieldsStorage = InternalStorage()
```

Comment:
SaveCardViewBindingHelper contains storage for fields. You can use it.

Reply:
Forget to remove, I moved `InternalStorage` to fragment, as some inputs should be added to internal storage only if it is visible or required by config type.